staging/src/k8s.io/apiserver/pkg/server/genericapiserver_graceful_termination_test.go

```diff
@@ -735,7 +735,8 @@ func setupInFlightWatchRequestHandler(s *GenericAPIServer) *inFlightRequest {
 			return
 		}
```

Comment:
maybe drop the select, delete the unreachable code, and change the above code to:
```
<-signals.ShuttingDown()
w.WriteHeader(http.StatusOK)
```

Reply:
Yes, that's simpler, if we agree that the error path was never meant to be executed. Changed.